src/EventTickets/resources/admin/components/EventDetailsPage/EventSection/EventSectionRowActions.tsx

```diff
@@ -10,31 +10,8 @@ const eventTicketsApi = new EventTicketsApi(apiSettings);
  */
 export function EventSectionRowActions({event, openEditModal}) {
     return () => {
-        const deleteItem = async (itemId) =>
```

Comment:
> which also no longer includes the delete action.

I would suggest submitting this as a separate change instead, which would make it easier to troubleshoot shoot in the future. By including this within another change the `git blame` will be confusing.

Reply:
Reverted the changes and created a new PR for that.